src/components/HomeCard/home-card.component.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+
+import Text from '../../utils/components/Text';
```

Comment:
All files are named with js extension. Why did you add a jsx extension instead? 
https://reactjs.org/docs/react-without-jsx.html